frontend/lib/ui/navbar.tsx

```diff
@@ -48,6 +59,15 @@ const NavbarContext = React.createContext<NavbarContext>({
   toggleDropdown: () => {},
 });
 
+export function getUserInitials({
+  firstName,
+  lastName,
+}: Pick<AllSessionInfo, "firstName" | "lastName">): string {
+  return [firstName, lastName]
+    .map((value) => (value && value[0].toUpperCase()) || "?")
```

Comment:
Just curious—is there a reason you aren't using a ternary operator here? Or is there a possibility that `value` is defined and `value[0]` isn't?

Reply:
Oh good point, using a ternary will be more readable here, I'll change it!